### ISSUE TITLE
Chore: Update documentation for Auto-WCAG workflow/process

### ISSUE THREAD

**Issue by annethyme** [auto-wcag/auto-wcag#178]
Things that probably should be mentioned in the documentation: 

- Automated project board: https://github.com/auto-wcag/auto-wcag/projects/4, and how to use it
- Workflow
        - To Do
        - In Progress
        - Needs Review
        - Reviewer Approved (3 approvals)
        - Done (For things to move into "Done", it needs to be "Reviewer Approved" and have been on the agenda of an Auto-WCAG meeting (visualised with labels, e.g. "july-agenda" for the Auto-WCAG July meeting).
- Label: "Reviewer wanted"
- Auto-WCAG Github guide
- Auto-WCAG Meetings - when, who and for what? - Label: "[month]-agenda"
- Relation between Auto-WCAG and WAI-Tools project

**Comment by jkodu** [comment]
@annethyme - believe this will be tackled by changes requested in - https://github.com/auto-wcag/auto-wcag/issues/183

If not please feel free to open.
